Find a functionally equivalent code:android.content.AsyncTaskLoader.LoadTask.doInBackground:COMMENT
Method Modifier: protected   
Comment:/* Runs on a worker thread */

@@ -1,10 +1,10 @@
 {
     if (DEBUG)
-        Slog.v(TAG, this + " >>> doInBackground");
+        Log.v(TAG, this + " >>> doInBackground");
     try {
         D data = AsyncTaskLoader.this.onLoadInBackground();
         if (DEBUG)
-            Slog.v(TAG, this + "  <<< doInBackground");
+            Log.v(TAG, this + "  <<< doInBackground");
         return data;
     } catch (OperationCanceledException ex) {
         if (!isCancelled()) {
@@ -12,18 +12,18 @@
             throw ex;
         }
         if (DEBUG)
-            Slog.v(TAG, this + "  <<< doInBackground (was canceled)", ex);
+            Log.v(TAG, this + "  <<< doInBackground (was canceled)", ex);
         return null;
     }
 }

